WMIAdventure/frontend/src/js/components/auth/molecules/AuthForm/AuthForm.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import Form from "./styled-components/Form";
+import Fieldset from "./styled-components/Fieldset";
+import Legend from "./styled-components/Legend";
+import Input from "./styled-components/Input";
+import Submit from "./styled-components/Submit";
+import Label from "./styled-components/Label";
+import InputContainer from "./styled-components/InputContainer";
+import ColumnGapContainer from "../../../global/molecules/ColumnGapContainer";
+import Line from "./styled-components/Line";
+import A from "./styled-components/A";
+import Div from "./styled-components/Div";
+
+class AuthForm extends React.Component {
```

Comment:
Brakuje tutaj komunikatów z błędami z walidacji backendu

Reply:
Lepiej aby osoba, która robiła na to osobny task, to zrobiła

Reply:
Ale osobny task jest na sam walidator a nie na komponent

Reply:
No ale w jakimś stopniu ten walidator będzie oddziaływał na ten komponent, jest to task po stronie Frontendu, To jest task od designu tylko. Skąd mam wiedzieć jak dokładnie będzie chciał zrobić to podmiot z tamtego taska? Dlaczego mam narzucać komuś chociażby nazwę propsa?